downloader.py

```diff
@@ -0,0 +1,41 @@
+import hashlib
+import os
+import re
+
+import requests
+
+
+DATASETS_LIST_URL = 'https://database.lichess.org/standard/list.txt'  # list of urls to all files with datasets
+CHECKSUMS_LIST_URL = 'https://database.lichess.org/standard/sha256sums.txt'  # list of sha256 checksums of data files
+
+
+def download_data(month: str) -> str:
+    """Function to download given dataset from lichess and save it
+
+    :param month: month of dataset among available in lichess.org in form `yyyy-mm`
+
+    :return: name of file containing downloaded data
+    """
+    if re.match(r'^\d{4}-(0[1-9]|1[0-2])$', month) is None:  # check for proper format
+        raise ValueError('Month parameter should be in form `yyyy-mm`')
+    datasets_list = '\n' + requests.get(DATASETS_LIST_URL).text + '\n'
```

Comment:
Just being curious, why these two '\n's?

Reply:
Later (line 23) I search in `datasets_list` for occurence of `month` and there's a condition for string to start and end with '\n'. If I haven't add these first and last line can't be found.